Source/Core/AST/GotoBoogie/Block.cs

```diff
@@ -7,37 +7,11 @@ namespace Microsoft.Boogie;
 
 public sealed class Block : Absy
 {
-  private string /*!*/ label; // Note, Label is mostly readonly, but it can change to the name of a nearby block during block coalescing and empty-block removal
+  public string /*!*/ Label { get; set; } // Note, Label is mostly readonly, but it can change to the name of a nearby block during block coalescing and empty-block removal
 
-  public string /*!*/ Label
-  {
-    get
-    {
-      Contract.Ensures(Contract.Result<string>() != null);
-      return this.label;
-    }
-    set
-    {
-      Contract.Requires(value != null);
-      this.label = value;
-    }
-  }
-
-  [Rep] [ElementsPeer] public List<Cmd> /*!*/ cmds;
-
-  public List<Cmd> /*!*/ Cmds
-  {
-    get
-    {
-      Contract.Ensures(Contract.Result<List<Cmd>>() != null);
-      return this.cmds;
-    }
-    set
-    {
-      Contract.Requires(value != null);
-      this.cmds = value;
-    }
-  }
+  [Rep] 
+  [ElementsPeer] 
+  public List<Cmd> /*!*/ Cmds;
```

Comment:
Better because simpler? Technically we're losing the contract.

Reply:
TBH I don't value the nullability contracts. It would be simple to add them by adding `#nullable enable`